<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>coffee_proj</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>90</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_init">
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_backup">
      <property name="text">
       <string>백업</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_restore">
      <property name="text">
       <string>복원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
